src/main/java/io/vertx/mqtt/messages/codes/MqttUnsubAckReasonCode.java

```diff
@@ -0,0 +1,22 @@
+package io.vertx.mqtt.messages.codes;
+
+public enum MqttUnsubAckReasonCode {
```

Comment:
why doesn't it extend `MqttReasonCode` like others ?

Reply:
Common handling wasn't needed in vertx-mqtt for this message. Now added that interface for uniformity.